src/std/net/socket/kqueue-server.ss

```diff
@@ -91,7 +91,7 @@ package: std/net/socket
                 (if io-out
                   (kqueue-kevent-add kq sock EVFILT_WRITE)
                   (begin
-                    (kqueue-kevent-del kq sock)
+                    #;(kqueue-kevent-del kq sock)
```

Comment:
is this no longer necessary? Or is this an optimization because the socket will be closed?